client/protocols/protocols.go

```diff
@@ -0,0 +1,11 @@
+package protocols
+
+import (
+	rewardspb "github.com/coinbase/staking-client-library-go/gen/go/coinbase/staking/rewards/v1"
```

Comment:
I think considering today's situation - we should put this in the rewards pkg as its not something common to both our APIs.

Reply:
Yeh I think that's a good callout. It'd be great to combine these client experiences over time, but will move these protocols to `rewards` for now.